### PR TITLE
Add link relation to index doc

### PR DESCRIPTION
**Pull request by NaokiTsuchiya**
`alps.link` element dump to index doc for providing a transition to the link from asd document.

image

### CODE REVIEW

**Comment by NaokiTsuchiya** [comment]
@koriym This PR is the end of my work in this weekend.

**Comment by koriym** [comment]
@NaokiTsuchiya Is `Relations` better than `LinkRelations` ?

File: src/IndexPage.php [hunk 1]
```diff
@@ -82,6 +82,19 @@ private function tags(array $tags): string
...
+    {
+        if ($linkRelations === null) {
+            return '';
+        }
+
+        if ((string) $linkRelations === '') {
+            return '';
+        }
+
+        return PHP_EOL . ' * LinkRelations' . PHP_EOL . $linkRelations;
```
Comment (koriym) [inline src/IndexPage.php:98]: Relationsに

**Review by koriym — CHANGES_REQUESTED** [review]

File: src/IndexPage.php (hunk 1 above)
Comment (NaokiTsuchiya) [reply src/IndexPage.php]: Fixed

**Pull request merged**